schemas/2015-01-01/deploymentTemplate.json

```diff
@@ -904,25 +904,25 @@
-                  { "$ref": "https://schema.management.azure.com/schemas/2015-03-01-preview/Microsoft.HDInsight.json#resourceDefinitions/clusters_extensions" },
-                  { "$ref": "https://schema.management.azure.com/schemas/2018-06-01-preview/Microsoft.HDInsight.json#resourceDefinitions/clusters" },
-                  { "$ref": "https://schema.management.azure.com/schemas/2018-06-01-preview/Microsoft.HDInsight.json#resourceDefinitions/clusters_applications" },
-                  { "$ref": "https://schema.management.azure.com/schemas/2018-06-01-preview/Microsoft.HDInsight.json#resourceDefinitions/clusters_extensions" },
-                  { "$ref": "https://schema.management.azure.com/schemas/2015-06-01-preview/Microsoft.Security.json#resourceDefinitions/jitNetworkAccessPolicies"},
-                  { "$ref": "https://schema.management.azure.com/schemas/2017-08-01-preview/Microsoft.Security.json#resourceDefinitions/pricings"},
-                  { "$ref": "https://schema.management.azure.com/schemas/2017-08-01-preview/Microsoft.Security.json#resourceDefinitions/securityContacts"},
-                  { "$ref": "https://schema.management.azure.com/schemas/2017-08-01-preview/Microsoft.Security.json#resourceDefinitions/workspaceSettings"},
-                  { "$ref": "https://schema.management.azure.com/schemas/2017-08-01-preview/Microsoft.Security.json#resourceDefinitions/autoProvisioningSettings"},
-                  { "$ref": "https://schema.management.azure.com/schemas/2017-08-01-preview/Microsoft.Security.json#resourceDefinitions/advancedThreatProtectionSettings"},
-                  { "$ref": "https://schema.management.azure.com/schemas/2017-08-01-preview/Microsoft.Security.json#resourceDefinitions/informationProtectionPolicies"},
-                  { "$ref": "https://schema.management.azure.com/schemas/2018-06-01/Microsoft.Security.json#resourceDefinitions/pricings"},
-                  { "$ref": "https://schema.management.azure.com/schemas/2018-06-01/Microsoft.Security.json#resourceDefinitions/securityContacts"},
-                  { "$ref": "https://schema.management.azure.com/schemas/2018-06-01/Microsoft.Security.json#resourceDefinitions/workspaceSettings"},
-                  { "$ref": "https://schema.management.azure.com/schemas/2018-06-01/Microsoft.Security.json#resourceDefinitions/autoProvisioningSettings"},
-                  { "$ref": "https://schema.management.azure.com/schemas/2018-06-01/Microsoft.Security.json#resourceDefinitions/advancedThreatProtectionSettings"},
-                  { "$ref": "https://schema.management.azure.com/schemas/2018-06-01/Microsoft.Security.json#resourceDefinitions/informationProtectionPolicies"},
+                  { "$ref": "https://schema.management.azure.com/schemas/2015-03-01-preview/Microsoft.HDInsight.json#/resourceDefinitions/clusters" },
```

Comment:
>/ [](start = 117, length = 1)

I assume this is the focus of the change?

Reply:
Yes the only change in each line is the new `/` after the `#`. Without it, a reference to an "id" is implied.